code/src/reducers/todos.js

```diff
@@ -0,0 +1,81 @@
+import { createSlice } from "@reduxjs/toolkit";
+import uniqid from "uniqid";
+
+const todos = createSlice({
```

Comment:
i like that you had comments on the components. maybe that should be applied here sine im like "impressive but what is happening"